core/src/test/scala/org/apache/spark/security/CryptoStreamUtilsSuite.scala

```diff
@@ -167,36 +164,6 @@ class CryptoStreamUtilsSuite extends SparkFunSuite {
     }
   }
 
-  test("error handling wrapper") {
```

Comment:
Do we need to remove this test case inevitably? I'm wondering if we can keep some test coverage to make it sure that CRYPTO-141 is fixed.

Reply:
This test only tests the wrapper class `ErrorHandlingReadableChannel`. As it is removed, this test has nothing to test.

Reply:
The original ticket [SPARK-25535](https://issues.apache.org/jira/browse/SPARK-25535) and the PR doesn't add test for CRYPTO-141...